WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt

```diff
@@ -1113,7 +1127,10 @@ d_juhuria = {
 	culture = juhuro
 	
 	allow = {
-		always = no
+ 		OR = {
+			culture = juhuro
+			religion = tzadikite
+		}
```

Comment:
What does that even mean? What is "dynamic dejure structure"?

If you're going to massively overhaul titles soon, I can wait on any of that and not mess with the titular duchies and the like until them.